packages/framework/src/Model/Product/ProductCachedAttributesFacade.php

```diff
@@ -81,7 +81,7 @@ public function getProductParameterValues(Product $product)
         $productParameterValues = $this->parameterRepository->getProductParameterValuesByProductSortedByName($product, $locale);
         foreach ($productParameterValues as $index => $productParameterValue) {
             $parameter = $productParameterValue->getParameter();
-            if ($parameter->getName() === null
+            if ($parameter->getName() === ''
```

Comment:
??????
```suggestion
            if ($parameter === null
```

Reply:
This wouldn't be true because we want to know if the parameter name is empty